operation.go

```diff
@@ -18,11 +18,17 @@ import (
 	"golang.org/x/tools/go/loader"
 )
 
+// RouteProperties describes HTTP properties of a single router comment.
+type RouteProperties struct {
+	HTTPMethod string
+	Path       string
+	IsDefault  bool
```

Comment:
I know a boolean doesn't eat too much memory, but you are not using it outside `ParseRouterComment` so please get rid of it.